security/uswds-2.13.1-zip-hash.txt

```diff
@@ -1 +1 @@
-b7806c11787ca7a2fb44c0dee791c150f2c4d4aeddc59103c811b3c45210d4a5
\ No newline at end of file
+c048e4958cbc51a5512d89e321bc7bb90aa3bec00539330042125d6b2c0a0c45
```

Comment:
Confirmed that the hash found in `develop` (c048e4958cbc51a5512d89e321bc7bb90aa3bec00539330042125d6b2c0a0c45) matches the zip file as well as the [release notes](https://github.com/uswds/uswds/releases/tag/v2.13.1) and therefore should be retained.